net/sshuttle/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2022 Thomas Cujé
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sshuttle
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=$(PKG_NAME)
+PKG_HASH:=f5a3ed1e5ab1213c7a6df860af41f1a903ab2cafbfef71f371acdcff21e69ee6
+
+PKG_MAINTAINER:=Thomas Cujé <dev.cuje@freenet.de>
+PKG_LICENSE:=LGPL-2.1-only
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/sshuttle
+  SUBMENU:=SSH
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=sshuttle: where transparent proxy meets VPN meets ssh
+  URL:=https://github.com/sshuttle/sshuttle
+  DEPENDS:= \
+  +python3-light \
+  +python3-ctypes \
+  +python3-urllib \
+  +iptables-mod-extra \
+  +iptables-mod-nat-extra \
+  +iptables-mod-ipopt
+endef
+
+define Package/sshuttle/description
+  Transparent proxy server that works as a poor man's VPN. Forwards over ssh. Doesn't require admin. Works with Linux and MacOS. Supports DNS tunneling.
+endef
+
+define Package/sshuttle/conffiles
+/etc/config/sshuttle
+endef
+
+define Package/sshuttle/install
```

Comment:
This is still wrong, because in the package is currently only init script and config.

Reply:
Sorry? I think the `fail2ban` package has the exact same instructions. What am I missing here?

Reply:
should use `Py3Package/sshuttle/install`, and please take care of executable files located under `$(PKG_INSTALL_DIR)/usr/bin`

Reply:
Ok, I think it's done with my last commit, please have a look.

Reply:
lgtm